### PR TITLE
Complete view following  the wireframes

### PR DESCRIPTION
**Pull request by Yacoubou-seidou**
### Complete view following the wireframes
- Implement the design from the sneak peek wireframes.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+            <li>
+              <h4><%= comment.user.name %>:</h4>
+              <p><%= comment.text[0..100]... %></p>
+            </li>
+          <% end %>
+        </ul>
+        <%= 'No comments yet!' if post.comments.empty? %>
+      </li>
+    <% end %>
+  </ul>
```
Comment (muneebulrehman) [inline app/views/posts/index.html.erb]: As per the project requirements, on the Posts Index page, if a post has any comments, it should utilize the `most_recent_comments` method in the posts model to fetch the recent comments. However, in your case you are not utilizing this method, instead, you are displaying all the comments belonging to a post. Kindly fix this issue. ⬇️
image

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Yacoubou-seidou , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Great job setting up the views
- Great job using the `latest_posts` method to fetch recent posts on the User show page.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] I would strongly recommend making the user index page the root page. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (Yacoubou-seidou) [reply app/views/posts/index.html.erb]: @muneebulrehman Thanks for the reviews !

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
```
Comment (Qoosim) [inline app/views/posts/index.html.erb:37]: - [ ] As mentioned by the previous reviewer, as per the project requirements, on the Posts Index page, if a post has any comments, it should utilize the most_recent_comments method in the posts model to fetch the 5 most_recent comments. However, in your case, this method does not work as expected, instead, it displays all the comments belonging to a post that I created (6 . You might need to look into the post model and check where the issue is. Kindly fix this issue.

### Comments error.

comment-error

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @Yacoubou-seidou, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (Yacoubou-seidou) [reply app/views/posts/index.html.erb:37]: @Qoosim Sorry but I fixed that you are not on the right page this page is the `/posts/:id` so all comments are there check the preview reviewer request is the path `/posts`. I can't find your name in the form dropdown, Kindly give me your full name so I can complete the form to ask for a second opinion thanks.
blogg

**Review by nicupop729 — COMMENTED** [review]
Hello @Yacoubou-seidou  👋,

_This is an additional review as a result of the second opinion request._

Congratulations for the hard work done in this milestone! :clap:

I noticed that after the last review you pushed some changes even thought you asked for a second opinion review. Please note that if you don't agree with a review and if you ask for a second opinion review, you should not push other changes, but wait for the review.

I see that now only the last 3 posts are displayed, and only the last 5 comments for each post. 
This is good, and the requirements mentioned by the last reviewer are fulfilled.  The project is ready to be approved!

As optional requirement, please take note of the next comment
- [ ] as the first reviewer mentioned, the user index page should be the root page. At the moment, the root page is not displaying the users, but an empty page:
image

Please ask for another review, and the next reviewer will be more than happy to approve your CRR!

Keep rocking 🚀!

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @Yacoubou-seidou, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Application views have been added correctly :white_check_mark: 
- [x] Application works as expected :white_check_mark: 
- [x] Git flow is used correctly :white_check_mark: 
- [x] Repository has a professional readme file :white_check_mark:     

## Optional suggestions
- [ Optional ] Please consider making the user `index` page the root page of your application. At the moment, the root page is not displaying the users, but an empty page: I would like to suggest that you add the code below in your `routes.rb` as the first route to achieve this.

## Example code
```ruby
  root "users#index"
```

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**